### PR TITLE
chore(digitaltwins-device): Pnp Refresh Bug bash updates

### PR DESCRIPTION
**Pull request by jebrando**
# Checklist
- [ ] I have read the [contribution guidelines] (https://github.com/Azure/azure-iot-sdk-node/blob/master/.github/CONTRIBUTING.md).
- [ ] I added or modified the existing tests to cover the change (we do not allow our test coverage to go down).
- If this is a modification that impacts the behavior of a public API
  - [ ] I edited the corresponding document in the `devdoc` folder and added or modified requirements.

# Reference/Link to the issue solved with this PR (if any)

# Description of the problem

# Description of the solution

### CODE REVIEW

File: digitaltwins/samples/service/typescript/invoke_component.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import { DigitalTwinServiceClient } from 'azure-iot-digitaltwins-service';
```
Comment (zolvarga) [inline digitaltwins/samples/service/typescript/invoke_component.ts:4]: I would suggest to use "invoke_component_command.ts" for a file name to be consistent with the JS sample.

**Comment by zolvarga** [comment]
>     console.log('invoking command ' + commandName + ' on component instance' + componentName + ' for device ' + deviceId + '...');

Add space after the "instance" for readability

---
Refers to: digitaltwins/samples/service/javascript/invoke_component_command.js:26 in cb4fca9. [](commit_id = cb4fca97e3f535a8df4e174b5f42b5a1eaed105f, deletion_comment = False)

**Comment by anthonyvercolano** [comment]
> {

Are all of these .json files copies of the ones "used/defined/accessed" by the c sdk?

---
Refers to: digitaltwins/samples/device/models/DeviceInformation.interface.json:1 in cb4fca9. [](commit_id = cb4fca97e3f535a8df4e174b5f42b5a1eaed105f, deletion_comment = False)

**Comment by anthonyvercolano** [comment]
>     console.log('invoking command ' + commandName + ' on component instance' + componentName + ' for device ' + deviceId + '...');

Actually I think instance is redundant. Components are by definition a particular interface instance.  So you could read this as " on interface instance instance".

---
Refers to: digitaltwins/samples/service/javascript/invoke_component_command.js:26 in cb4fca9. [](commit_id = cb4fca97e3f535a8df4e174b5f42b5a1eaed105f, deletion_comment = False)

File: digitaltwins/samples/service/typescript/invoke_component.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+const deviceId = process.env.IOTHUB_DEVICE_ID || '';
+const connectionString = process.env.IOTHUB_CONNECTION_STRING || '';
+const componentName = process.env.IOTHUB_COMPONENT_NAME || 'sensor'; // for the environmental sensor, try "sensor"
+const commandName = process.env.IOTHUB_COMMAND_NAME || 'turnon'; // for the environmental sensor, you can try "blink", "turnoff" or "turnon"
+const commandPayload = process.env.IOTHUB_COMMAND_PAYLOAD || ''; // for the environmental sensor, it really doesn't matter. any string will do.
+
+let credentials = new IoTHubTokenCredentials(connectionString);
+let digitalTwinServiceClient = new DigitalTwinServiceClient(credentials);
+
+console.log('invoking command ' + commandName + ' on component instance' + componentName + ' for device ' + deviceId + '...');
```
Comment (anthonyvercolano) [inline digitaltwins/samples/service/typescript/invoke_component.ts:16]: > instance [](start = 62, length = 9)

And I'll say remove the word instance.

**Comment by anthonyvercolano** [comment]
>     "azure-iothub": "1.12.0-pnp-refresh.0",

Why do you need this?

---
Refers to: digitaltwins/samples/service/javascript/package.json:23 in cb4fca9. [](commit_id = cb4fca97e3f535a8df4e174b5f42b5a1eaed105f, deletion_comment = False)

File: digitaltwins/samples/service/typescript/package.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  "version": "1.0.0-pnp-refresh.0",
+  "description": "Azure IoT SDK - Digital Twin Client typescript samples",
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "dependencies": {
+    "azure-iot-device": "1.12.0-pnp-refresh.0",
+    "azure-iot-common": "1.12.0-pnp-refresh.0",
+    "azure-iot-device-mqtt": "1.12.0-pnp-refresh.0",
+    "azure-iot-digitaltwins-device": "1.0.0-pnp-refresh.0",
+    "azure-iot-digitaltwins-service": "1.0.0-pnp-refresh.0"
```
Comment (anthonyvercolano) [inline digitaltwins/samples/service/typescript/package.json:12]: Why do you need these?

**Comment by jebrando** [comment]
> {

Yes, these are from the plug and play repo

---
In reply to: [628248408](https://github.com/Azure/azure-iot-sdk-node/pull/798#issuecomment-628248408) [](ancestors = 628248408)

---
Refers to: digitaltwins/samples/device/models/DeviceInformation.interface.json:1 in cb4fca9. [](commit_id = cb4fca97e3f535a8df4e174b5f42b5a1eaed105f, deletion_comment = False)

**Comment by jebrando** [comment]
>     "azure-iothub": "1.12.0-pnp-refresh.0",

I do not, will remove

---
In reply to: [628251596](https://github.com/Azure/azure-iot-sdk-node/pull/798#issuecomment-628251596) [](ancestors = 628251596)

---
Refers to: digitaltwins/samples/service/package.json:23 in cb4fca9. [](commit_id = cb4fca97e3f535a8df4e174b5f42b5a1eaed105f, deletion_comment = False)

**Comment by anthonyvercolano** [comment]
>     "@azure/event-hubs": "^2.1.1"

Why do you need this?

---
Refers to: digitaltwins/samples/service/javascript/package.json:24 in cb4fca9. [](commit_id = cb4fca97e3f535a8df4e174b5f42b5a1eaed105f, deletion_comment = False)

File: digitaltwins/samples/service/typescript/dist/invoke_component.d.ts [hunk 4]
```diff
@@ -0,0 +1,2 @@
+export {};
+//# sourceMappingURL=invoke_component.d.ts.map
```
Comment (anthonyvercolano) [inline digitaltwins/samples/service/typescript/dist/invoke_component.d.ts:2]: Delete.  Dist directory is not to be checked in.

**Pull request merged**